scripts/analyze-bundle/command.mjs

```diff
@@ -0,0 +1,172 @@
+import fs from 'fs/promises';
```

Comment:
Please follow other commands convention and move all the functions out of `command.js`. This file should only contains a function called `run`.  You can take a look at `scripts/publish/` as an example.